code/game/objects/items/toys.dm

```diff
@@ -494,22 +503,65 @@
 			if(uses)
 				uses--
 				if(!uses)
-					user << "<span class='danger'>You used up your crayon!</span>"
+					user << "<span class='danger'>You used up your [src]!</span>"
 					qdel(src)
 	return
 
 /obj/item/toy/crayon/attack(mob/M as mob, mob/user as mob)
 	if(M == user)
-		user << "You take a bite of the crayon. Delicious!"
+		user << "You take a bite of the [src]. Delicious!"

```

Comment:
I left it in because I imagine it more like huffing the paint, delicious, fume inducing paint.